man/mustache.5.ron

```diff
@@ -146,32 +250,54 @@ Hash:
 
 Output:
 
-    <b>resque</b>
-    <b>hub</b>
-    <b>rip</b>
+      <b>resque</b>
+      <b>hub</b>
+      <b>rip</b>
+
+The same effect as above can be obtained without nested objects, by using
+the implicit iterator (see **Variables** above).
+
+Template:
+
+    {{#repo}}
+      <b>{{.}}</b>
+    {{/repo}}
+
+Hash:
+
+    {
+      "repo": ["resque", "hub", "rip"]
+    }
+
+Output:
+
+      <b>resque</b>
+      <b>hub</b>
+      <b>rip</b>
 
 **Lambdas**
 
-When the value is a callable object, such as a function or lambda, the
-object will be invoked and passed the block of text. The text passed
-is the literal block, unrendered. `{{tags}}` will not have been expanded
-- the lambda should do that on its own. In this way you can implement
-filters or caching.
+When any value found during the lookup is a callable object, such as a
+function or lambda, the object will be invoked and passed the block of
+text. The text passed is the literal block, unrendered. `{{tags}}` will
+not have been expanded.
+
+An **optional** part of the specification states that if the final key in
```

Comment:
Comment above applies here as well (the entire Lambda spec is optional, right?).